contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -255,6 +265,34 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     }
   }
 
+  /// @notice Updates nodes.  The node admin can update node signer addresses
+  /// and reconfigure the node's supported capabilities
+  /// @param nodes The nodes to update
+  function updateNodes(Node[] calldata nodes) external {
+    for (uint256 i; i < nodes.length; ++i) {
+      Node memory node = nodes[i];
+
+      NodeOperator memory nodeOperator = s_nodeOperators[node.nodeOperatorId];
+      if (msg.sender != nodeOperator.admin) revert AccessForbidden();
```

Comment:
We should allow owner's to do this, too (I forgot it in addNodes as well). This is because we might be doing the initial setup to save time (for us) and costs (for NOPs).